src/index.html

```diff
@@ -6,8 +6,28 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
+  <header class="header">
+    <a class="img" href="http://localhost:8080/logo.6023b87e.png"><img  src="./images/logo.png" alt="logo"></a>
+
+    <nav class="nav">
+      <button class="button"></button>
+      <ul class="nav__list">
+        <li><a class="nav__link is-active" href="#">Apple</a></li>
+        <li><a class="nav__link" href="#">Samsung</a></li>
+        <li><a class="nav__link nav__link__tree" href="#">Smartphones</a></li>
+        <li data-qa="hover"><a class=" nav__link nav__link__four" href="#">Laptops & Computers</a></li>
+        <li><a class="nav__link nav__link__five" href="#">Gadgets</a></li>
+        <li><a class="nav__link" href="#">Tablets</a></li>
+        <li><a class="nav__link" href="#">Photo</a></li>
+        <li><a class="nav__link link-last" href="#">Video</a></li>
+      </ul>
```

Comment:
not fixed:
![image](https://github.com/mate-academy/layout_moyo-header/assets/86923089/b4724e3d-0bb6-46d3-bfc3-563a47b37f1c)